.vscode/settings.json

```diff
@@ -10,6 +10,9 @@
   "files.trimFinalNewlines": true,
   "files.insertFinalNewline": true,
   "editor.detectIndentation": false,
-  "editor.tabSize": 2,
-  "editor.insertSpaces": true
+  "editor.tabSize": 4,
```

Comment:
I'm a little nervous this is going to cause a bunch of whitespace changes in PRs that otherwise would be making minor changes. Do you mind talking me through the idea behind changing this setting?

Reply:
I was actually trying to prevent exactly the scenario you're worried about.  I work in vscode, and`"editor.tabSize": 2` cause new `.cs` files - and any file I make changes to - to reformat to 2-space indentation.  All the `.cs` are indented using 4 spaces.

Good point though, I just pushed a change that only updates this setting for C#, so config files, scripts, etc. aren't affected.